modules/dnn/test/test_caffe_importer.cpp

```diff
@@ -55,6 +55,43 @@ static std::string _tf(TString filename)
     return (getOpenCVExtraDir() + "/dnn/") + filename;
 }
 
+static bool readFileInMemory(const string& filename, string& content)
```

Comment:
Do you mind if we replace it to `test_common.hpp` to reduce duplicated code? (see `test_tf_importer.cpp`)

Reply:
No problem. I was looking for this kind of common file and did not find it